easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-foss-2017b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '5.0.0'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+checksums = ['e35dcbae6084adcf4abb32514127c5eabd7d63b733852ccdb31e06f1373136da']  # llvm-5.0.0.src.tar.xz
```

Comment:
no need for trailing comment, should be 'glued' to `sources` line, empty line afterwards